pyugrid/ugrid.py

```diff
@@ -850,12 +850,14 @@ def save_as_netcdf(self, filepath):
             if self.edges is not None:
                 # Attribute required if variables will be defined on edges.
                 mesh.edge_node_connectivity = mesh_name + "_edge_nodes"
+                mesh.edge_dimension = mesh_name + "_num_edge"
```

Comment:
according to the standard:

"""
*The “Optionally required” attributes face_dimension and  edge_dimension are required only if the dimension ordering is nonstandard in any of the connectivity variables for faces and edges, respectively.
"""
is pyugrid ever going to save out dimension ordering in non-standard way?

I'm pretty sure that all the internal code is assuming it's standard, so probably not.